gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilSubplugin.kt

```diff
@@ -39,12 +39,23 @@ class AnvilSubplugin : KotlinGradleSubplugin<AbstractCompile> {
     // for this specific compile task will be included in the task output. The output of different
     // compile tasks shouldn't be mixed.
     val srcGenDir = File(project.buildDir, "anvil/src-gen-${kotlinCompile.name}")
```

Comment:
nit: use `File.separator` for platform-agnostic file-path building.

Reply:
You or someone else pointed this out already and then I forgot to fix it. This is a good PR to fix it.

Reply:
It was me 😄

Reply:
I feel bad